MUI_TimerBars/MUI_TimerBars.lua

```diff
@@ -678,6 +679,11 @@ do
 
                     ---@param bar TimerBar
                     for i, bar in ipairs(data.activeBars) do
+						auraInfo = GetAuraInfoByAuraID(self.field:GetUnitID(), bar.auraId, bar.AuraType);
```

Comment:
This needed to be data.frame because it is a private property (not a public one)